src/script/components/MessagesList/Message/VerificationMessage.tsx

```diff
@@ -88,7 +88,9 @@ const VerificationMessage: React.FC<VerificationMessageProps> = ({message}) => {
           <>
             <span className="message-header-plain-sender-name">{nameList}</span>
             <span>
-              {hasMultipleUsers ? t('conversationDeviceStartedUsingMany') : t('conversationDeviceStartedUsingOne')}
+              {hasMultipleUsers || isSelfClient
```

Comment:
sounds like strictly a German hack no?

Reply:
Any language with a plural form that's the same as a proper form

Reply:
It's possible that an extra translation would be better

Reply:
yeah, there's more than 2 case here, we need you/many/one again I guess.
you formal is 3rd person plural in German, but 3rd person singular in Spanish, 2nd plural in French etc...

Reply:
ive used the same format as the other translation location for now. Ill align with Astrid